Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/POMNavPage.xaml.cs

```diff
@@ -41,30 +86,74 @@ public TreeView1 TreeView
             get { return xTreeView; }
         }
 
-        public POMNavPage(string itemTypeName, eImageType itemTypeIcon, ITreeViewItem itemTypeRootNode, RoutedEventHandler saveAllHandler = null, RoutedEventHandler addHandler = null, EventHandler treeItemDoubleClickHandler = null)
+        public POMNavPage(Context context, string itemTypeName, eImageType itemTypeIcon, ITreeViewItem itemTypeRootNode, RoutedEventHandler saveAllHandler = null, RoutedEventHandler addHandler = null, EventHandler treeItemDoubleClickHandler = null)
         {
             InitializeComponent();
 
+            mContext = context;
```

Comment:
also here i think you needed to use SingleItemTreeViewSelectionPage unless i'm missing something, let's talk on it